Update cookbook example instructions (#146)
agno-agi/agno-docs: get-started/monitoring.mdx

title: Agent Observability
---

> ## Monitoring Your Agents

Reviewer: It should either be `Monitor your Agents` or `Monitoring`. I prefer the latter.

Reviewer 2: Thinking about it, I don't like either. You aren't really monitoring anything. You are saving/logging runs.
But if we stick to the wording, I prefer "Monitor your agents"

Reviewer: In our case here Monitoring / Observability are keywords that indicate that users can "track" their Agent runs. In my opinion, we have to use one of those keywords

Reviewer 3: "Monitor your Agents" is clear action-oriented directive let's go with that.

> Monitoring helps you track your agents' performance and session activity, ensuring everything runs smoothly. Agno provides built-in monitoring tools that allow you to view session data through [app.agno.com](https://app.agno.com).

Reviewer: ```suggestion
You can monitor your agents' performance and session activity to ensure smooth operation. Agno offers built-in monitoring capabilities, allowing you to track session and performance metrics through [app.agno.com](https://app.agno.com).
```

## Setup Process

### Step 1: Authenticate

Before enabling monitoring, you need to authenticate using one of these methods:

**Method A: Quick Setup**
```bash

> agno setup

Reviewer 2: ```suggestion
ag setup
```

```

> **Method B: Manual API Key Configuration**

Reviewer: ```suggestion
**Method B: API Key Configuration**
```

> **Method B: Manual API Key Configuration**

Reviewer 2: Where do you get this? We should probably say

Reviewer 3: Now that I think about it, there's no easy way to directly get the API key from the Agno app. Users first have to go to the app, click on the API key option in the sidebar, and then get the key. I am thinking if we can make this process easier for them in the app, like some sort of link that can directly open the API key dialog box for them, which will make it easier for them to copy and paste the API key. Regardless, it's a different problem to solve, but I want to know your thoughts on this @ysolanky @dirkbrnd.

Reviewer: @Ansub I think we do make it really easy for users to find the API key. They can access it right after they login on the dashboard page

Reviewer: Maybe a dedicated page for the key? But the key would still be one click away as it is right now

Reviewer 3: thinking if there can be a route like `app.agno.com/#api-key` which will open up the dialog box.  better DX imo.
